blockchain.go

```diff
@@ -397,14 +412,30 @@ func bootstrapLedger(
 		Weight:    fvm.AccountKeyWeightThreshold,
 	}
 
-	err := vm.Run(ctx, fvm.Bootstrap(flowAccountKey, genesisTokenSupply), ledger)
+	bootstrap := configureBootstrapProcedure(ctx, flowAccountKey, genesisTokenSupply)
+
+	err := vm.Run(ctx, bootstrap, ledger)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
+func configureBootstrapProcedure(ctx fvm.Context, flowAccountKey flowgo.AccountPublicKey, supply cadence.UFix64) *fvm.BootstrapProcedure {
+	if ctx.LimitAccountStorage {
+		return fvm.Bootstrap(flowAccountKey,
+			fvm.WithInitialTokenSupply(supply),
+			fvm.WithAccountCreationFee(10000000),
+			fvm.WithMinimumStorageReservation(10000000),
```

Comment:
Ideally these should match the values used on real networks by default, with the option to maybe configure them (like the computation limit). Maybe declare in and re-use constants from FVM

Reply:
done